website/pages/en/support.js

```diff
@@ -0,0 +1,135 @@
+| [0.64](https://www.npmjs.com/package/react-native-windows/v/v0.64-stable) | [Maintenance](#maintenance-support) | 03/15/2021 | *N/A*[¹](#1)| *N/A*[¹](#1) | 06/30/2022[²](#2) |
+| [0.63](https://www.npmjs.com/package/react-native-windows/v/v0.63-stable) | [Unsupported](#unsupported) | 01/24/2022 | *N/A*[¹](#1) | *N/A*[¹](#1) | 10/11/2021 |
+
+<div class="footnote">
+
+**Note:** All prior releases not listed are [Unsupported](#unsupported).
+
+<a name="1">¹</a>: These releases were made prior to the institution of this official support policy document.
+
+<a name="1">²</a>: These releases were already receiving [Maintenance](#maintenance-support) equivalent support, and the new "End of Support" dates were set to give users some transition time before they become officially [Unsupported](#unsupported).
+
+</div>
+
+## Summary
+
+RNW stable versions are referred to by their major and minor version number, i.e. **0.64**, mapping to the same React Native stable version, and are sourced from an equivalently named \`*-stable\` branch in the RNW repo, i.e. [0.64-stable](https://github.com/Microsoft/react-native-windows/tree/0.64-stable).
+
+A new stable version's life begins when that stable branch is created and a package published to NPM under the [preview tag](https://www.npmjs.com/package/react-native-windows/v/preview). That version is said to be *in preview* and has entered [Preview Support](#preview-support). After the preview period ends, the stable version becomes the new "latest" release and its package is published to NPM under the [latest tag](https://www.npmjs.com/package/react-native-windows/v/latest). That version has now entered [Active Support](#active-support).
+
+At this point, the previous latest stable version is now considered a "legacy" version. Such legacy versions will receive progressively reduced support over time, specifically:
+
+1. One final month of [Active Support](#active-support), followed by
+2. Two months of [Maintenance Support](#maintenance-support)
+
+After which, the legacy version will become [Unsupported](#unsupported).
+
+See below for further details on each support phase. For further details on the release process, see [RNW Versioning and Release Process](https://github.com/microsoft/react-native-windows/wiki/Versioning-and-Release-Process).
+
+## Support Phases
+
+### Active Support
+
+A stable version of RNW enters *Active Support* status as soon as the first official, non-preview, package of that stable version is published to NPM. This support continues with each patch release within that stable version.
+
+It is expected that the stable release branch is "stable" and will not take breaking API or template changes (relative to the first official release in that branch).
+
+The RNW Team commits to providing support for fixing issues that do not require breaking changes, including but not limited to:
+
+- Security and/or legal issues
+- Build and/or tooling issues
```

Comment:
do we want to mention "regressions in" here? 
e.g. VS2022/vc143 support is a build/tooling issue, but it's never worked before.

Reply:
mmm "precedent" :) I'd word all of these as aspirational and non-binding